dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/mbeans/PluginGenerator.java

```diff
@@ -866,18 +866,27 @@ protected synchronized void generateXML(String rootLoc, String serverName,
                     }
                     return;
                 }
+                
                 // Verify that the temp plugin file exists
                 if (!outFile.exists()) {
                     throw new FileNotFoundException("File " + outFile.asFile().getAbsolutePath() + " could not be found");
                 }
                 // Construct the actual plugin file path
                 File pluginFile = new File(outFile.asFile().getParentFile(), pcd.PluginConfigFileName);
+                
 
                 if (pluginFile.exists()) {
                     FileUtils.forceDelete(pluginFile);
                 }
 
-                outFile.asFile().renameTo(pluginFile);
+                boolean result = outFile.asFile().renameTo(pluginFile);
+
+                //See if rename was unsuccessful. 
+                if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled()) {
+                    if(!result){
```

Comment:
Shouldn't this be `tc.isDebugEnabled()`?

Reply:
It should. I am better than this.